packages/cozy-stack-client/src/OAuthClient.js

```diff
@@ -389,6 +389,15 @@ class OAuthClient extends CozyStackClient {
     this.oauthOptions.clientID = ''
   }
 
+  /**
+   * Reset the current OAuth client
+   */
+  resetClient() {
+    this.resetClientId()
+    this.setUri('')
```

Comment:
Nit: I prefer null in this cases.

Reply:
I had to edit `normalizeUri`. Is it better now?